accessibility-oscilloscope/configs/rpi4_defconfig

```diff
@@ -29,8 +30,7 @@ BR2_TARGET_ROOTFS_EXT2_SIZE="120M"
 BR2_PACKAGE_HOST_DOSFSTOOLS=y
 BR2_PACKAGE_HOST_GENIMAGE=y
 BR2_PACKAGE_HOST_MTOOLS=y
+BR2_PACKAGE_ASCII_CONV=y
 BR2_PACKAGE_DRV2506_CLEAN=y
-BR2_PACKAGE_OSCILLOSCOPE_DUMP=y
 BR2_PACKAGE_PNT_LUT=y
 BR2_PACKAGE_SIGNAL_COMPILER=y
-BR2_PACKAGE_ASCII_CONV=y
```

Comment:
BR2_PACKAGE_SIGNAL_PROCESSING=y
BR2_PACKAGE_OSCILLOSCOPE_TRIGGER=y
should be added to end of this as well